Sources/BraintreeCore/BTHTTP.swift

```diff
@@ -446,8 +446,12 @@ class BTHTTP: NSObject, URLSessionTaskDelegate {
         let json = try? JSONSerialization.jsonObject(with: data)
         let body = BTJSON(value: json)
         
-        guard let mutationName = body["operationName"].asString() else { return nil }
```

Comment:
The card GraphQL request is the only flow that uses `operationName` so for other flows such as Venmo we were sending the endpoint as `/graphql`. This update allows us to now send the expected query/mutation name.

Reply:
just curious - would this change affect us being able to fetch the mutation name for the card graphql flow 👀

Reply:
We still pass in a mutation as expected for the card flow, we just extract it from here: https://github.com/braintree/braintree_ios/blob/7b81129478fb3302b2ffacdbb4ade80e561faef7/Sources/BraintreeCard/BTCard.swift#L260 vs concatenating the mutation string to the `operationName`. So card is passed as `mutation TokenizeCreditCard` same as prior to this change.